packages/react/src/card/Card.tsx

```diff
@@ -19,6 +19,21 @@ const cardVariants = cva({
   },
 });
 
+const cardMediaVariants = cva({
+  base: [
+    // Hide overflow (outside the border radius)
+    'overflow-hidden',
+    // Bleed to the edges of the card (over the border)
```

Comment:
Grunnen til at `overflow-hidden` er satt her og ikke på selve kortet er at vi kommer til å trenge å støtte outline på kortet for fokus-markering